packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -681,3 +678,67 @@ function getDotDevDotVarsContent(
 
 	return null;
 }
+
+/**
+ * Returns a function that checks if a given resolved path to a `changedFile` matches
+ * some of the files that Cloudflare cares about (and that require the vite server to
+ * reload)
+ *
+ * Note: the function is cached based on the resolvedPluginConfig just so that
+ *       the extra paths resolutions are only done once
```

Comment:
arguably if I have to do this I could as well resolve all the paths and drop them onto `resolvedPluginConfig` when we create the object 🤔

I can give that alternative a shot if people dislike this approach 🙂